infra/modular-examples/aws-google-workspace/main.tf

```diff
@@ -250,3 +250,30 @@ module "psoxy-bulk" {
     IS_DEVELOPMENT_MODE = contains(var.non_production_connectors, each.key)
   }
 }
+
+
+module "psoxy_lookup_tables_builders" {
+  for_each = var.lookup_table_builders
+
+  source = "../../modules/aws-psoxy-bulk-existing"
+  # source = "git::https://github.com/worklytics/psoxy//infra/modules/aws-psoxy-bulk?ref=v0.4.7"
```

Comment:
module name is different `existing`, remember to change that in the git version when released